Makefile

```diff
@@ -113,59 +113,64 @@ manifests-gen-crds: controller-gen yq
 	for YAMLFILE in config/crd/bases/actions*.yaml; do \
 		$(YQ) '.spec.preserveUnknownFields = false' --inplace "$$YAMLFILE" ; \
 	done
+	make manifests-gen-crds-fix DELETE_KEY=x-kubernetes-list-type
```

Comment:
Note that we have already been removing `x-kubernetes-list-type` fields since https://github.com/actions/actions-runner-controller/pull/2132, and this is just another way to do it.